### ISSUE TITLE
Speichern sollte während des Sync-Vorgangs möglich sein

### ISSUE THREAD

**Issue by Ninelpienel** [BastianGanze/weebsync#9]
Aktuell ist es nicht möglich, einen Sync-Eintrag zu modifizieren, während weebsync etwas runterlädt. Es wäre toll, wenn das in Zukunft möglich wäre.

**Comment by BastianGanze** [comment]
Bitte mal näher erläutern was das erwartete Verhalten ist. Was, wenn man den derzeitigen Eintrag der gerade heruntergeladen wird so anpasst, dass der Download hinfällig wird oder der Dateiname sich ändert. Soll der sync mit der alten config durchlaufen und dann erst beim nächsten mal die neue konfiguration nehmen?

**Comment by Ninelpienel** [comment]
> Bitte mal näher erläutern was das erwartete Verhalten ist. Was, wenn man den derzeitigen Eintrag der gerade heruntergeladen wird so anpasst, dass der Download hinfällig wird oder der Dateiname sich ändert. Soll der sync mit der alten config durchlaufen und dann erst beim nächsten mal die neue konfiguration nehmen?

Ja, am besten so, wie du beschrieben hast. Mir geht es primär darum, dass ich neue Einträge hinzufügen und diese dann speichern kann, während Dateien geladen werden.
